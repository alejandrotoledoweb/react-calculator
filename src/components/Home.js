import React from 'react';

import '../index.css';

const Home = () => (
  <div className="quote">
    <h2>Welcome to my page!</h2>
    <p>This is my react app built to recreate a calculator. Try it!</p>
    <p>This page is part of the Projects in the Microverse Program.</p>
    <p>Visit My Github account to see more of my project</p>
    <a href="https://github.com/alejandrotoledoweb/react-calculator" target="_blank" rel="noreferrer">Alejandro Toledo</a>
  </div>
);

export default Home;
