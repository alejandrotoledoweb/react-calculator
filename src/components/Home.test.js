import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('it test the render Home page', () => {
  const home = renderer
    .create(<Home />)
    .toJSON();
  expect(home).toMatchSnapshot();
});
