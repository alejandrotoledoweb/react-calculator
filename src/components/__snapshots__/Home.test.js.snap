// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it test the render Home page 1`] = `
<div
  className="quote"
>
  <h2>
    Welcome to my page!
  </h2>
  <p>
    This is my react app built to recreate a calculator. Try it!
  </p>
  <p>
    This page is part of the Projects in the Microverse Program.
  </p>
  <p>
    Visit My Github account to see more of my project
  </p>
  <a
    href="https://github.com/alejandrotoledoweb/react-calculator"
    rel="noreferrer"
    target="_blank"
  >
    Alejandro Toledo
  </a>
</div>
`;
